src/Services/Account/Command/Infrastructure.EventStore/DependencyInjection/Options/SqlServerRetryingOptions.cs

```diff
@@ -4,7 +4,7 @@ namespace Infrastructure.EventStore.DependencyInjection.Options;
 
 public record SqlServerRetryOptions
 {
-    [Required, Range(5, 20)] public int MaxRetryCount { get; init; }
-    [Required, Timestamp] public TimeSpan MaxRetryDelay { get; init; }
+    [Required, Range(5, 20)] public  required int MaxRetryCount { get; init; }
```

Comment:
if this field has "required modifier" then not required anotation **Required**.
![image](https://user-images.githubusercontent.com/17841549/214350610-760176e4-acda-4008-aafa-45ac426d019a.png)